packages/mdc-chips/_mixins.scss

```diff
@@ -169,26 +170,20 @@
 }
 
 @mixin mdc-chip-leading-icon-margin(
-  $top: $mdc-chip-leading-icon-margin-top,
   $right: $mdc-chip-leading-icon-margin-right,
```

Comment:
Update these parameter names too.

Reply:
The function is called "margin". Isn't making the params also include "margin" redundant?

Reply:
margin suffix helps discover param type & improves readability when this variable used within mixin. But I don't have very strong preference on this once since these mixins used without named arguments most of the cases anyway `mdc-chip-leading-icon-margin(5px, 5px);`